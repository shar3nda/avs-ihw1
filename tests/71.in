10
2 6 2 8 6 6 7 1 -1 1