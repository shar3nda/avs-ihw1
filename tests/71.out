-4 1 -9 6 1 10 -6 -1 -7 -6